src/ehttpc.appup.src

```diff
@@ -0,0 +1,15 @@
+%% -*-: erlang -*-
+{VSN,
```

Comment:
I think explicitly specify the Target vsn better than VSN placeholder.

( VSN is a little trick to automatically set the target version based on the tag, but that's what the plugins in the [relup_helper](https://github.com/emqx/relup_helper) do. So it doesn't make sense to use this placeholder here )